### DISCUSSION TITLE
Can someone help me with implementing web3 wallet connection to my express js project

### DISCUSSION THREAD

**Discussion by egetelli** [WalletConnect/walletconnect-monorepo#1551] [Developer Support]
In src folder we have walletcnct.html and server.js for wallet connection. Html works i can see connect wallet button but when i click the button it doesn't work and i cannot connect my wallet. Can someone please look at my project ?
https://github.com/LelonDelonMelon/bitirme2/tree/master/backend

I have an error named Failed to load resource: the server responded with a status of 404 (Not Found)
And another error: Refused to execute script from 'http://localhost:8080/src/server.js' because its MIME type ('text/html') is not executable, and strict MIME type checking is enabled.

**Comment by xzilja** [comment]
@egetelli that example was outdated and remved recently. There is new html example based on vite. Server.js example was there only to serve files to html, thats basically it.
